.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+        exclude: docs/
+    -   id: check-yaml
+    -   id: check-added-large-files
+-   repo: https://gitlab.com/pycqa/flake8.git
+    rev: 3.9.2
+    hooks:
+    -   id: flake8
+        args: [--max-line-length, "100"]
+-   repo: https://github.com/ambv/black
+    rev: 22.6.0
+    hooks:
+    -   id: black
+        args: [--safe, --quiet, --line-length, "100"]
```

Comment:
Suggestion: add Python import sorting plugin. Two popular options are:

* [isort](https://pycqa.github.io/isort/docs/configuration/pre-commit.html)
* [reorder_python_imports](https://github.com/asottile/reorder_python_imports#as-a-pre-commit-hook)

reorder_python_imports forces each import to be in separate line, i.e. it will turn this:

```
from typing import Literal, Union
```

into this:

```
from typing import Literal
from typing import Union
```

The reason is that this style reduces number of merge conflicts. It also causes large import block at top of files, which some might consider ugly. Personally, I got used to this style.

Reply:
I ran `isort` and its result conflicts with flake8 and black. I had no issue by running `reorder_python_imports`, although it is very verbose, but it may result in less merge conflicts. So let's choose `reorder_python_imports` and see.

Reply:
Thanks for the information @bruno-fs! Well, let's go with `reorder_python_imports` in Camayoc.